### ISSUE TITLE
unable to play danger without local command

### ISSUE THREAD

**Issue by sylvek** [danger/danger#489]
hello,
thank you for your app. it seems to give huge promesses.
i spend my day to learn more about it and after reading the 44 issues and the website i'm missing something…

when i play "bundle exec danger local", that works as expected.
but when i play "bundle exec danger" i receive this stacktrace

```
> $ bundle exec danger              
bundler: failed to load command: danger (/Users/sylvek/.rbenv/versions/2.0.0-p648/bin/danger)
NameError: undefined local variable or method `ci_klass' for #<Danger::Executor:0x007f9af2202f80>
  /Users/sylvek/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/danger-3.0.3/lib/danger/danger_core/executor.rb:16:in `run'
  /Users/sylvek/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/danger-3.0.3/lib/danger/commands/runner.rb:56:in `run'
  /Users/sylvek/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/claide-1.0.0/lib/claide/command.rb:334:in `run'
  /Users/sylvek/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/danger-3.0.3/bin/danger:5:in `<top (required)>'
  /Users/sylvek/.rbenv/versions/2.0.0-p648/bin/danger:22:in `load'
  /Users/sylvek/.rbenv/versions/2.0.0-p648/bin/danger:22:in `<top (required)>'
```

i didn't find explanation about the workflow of "Danger". i read that "she" try to discover if the platform is a CI but how?

**Comment by orta** [comment]
Your computer won't be set up in the way that a CI server is set up, Danger looks through the environment variables of the current shell to determine which CI she is running on. I've improved the docs there, and the crash you've experienced was fixed in https://github.com/danger/danger/pull/484 - I'm gonna ship this today, which should fix that problem

**Comment by sylvek** [comment]
Ok great! Where may I find this doc?

**Comment by orta** [comment]
https://github.com/danger/danger.systems/pull/104 has the changes, it's not been merged yet though, waiting on the 3.1 releases

**Comment by orta** [comment]
Alright that's everything shipped 👍
